metaflow/datatools/s3.py

```diff
@@ -37,6 +39,19 @@
 except:
     boto_found = False
 
+def ensure_unicode(x):
+    return None if x is None else to_unicode(x)
+
+S3GetObject = namedtuple_with_defaults(
+    'S3GetObject', 'key offset length')
+
+S3PutObject = namedtuple_with_defaults(
+    'S3PutObject', 'key value content_type metadata')
```

Comment:
let's add `path` here too so the fields make more sense for `put_files`

Reply:
Done